livnote/frontend/desktop/components/notes/documentUtils.ts

```
@@ -81,35 +161,182 @@ export function generatePreview(fullNote: Note, maxNodes: number = 3): NotePrevi
+
+  return content.map(node => nodeToHTML(node)).join('');
+}
+
+function nodeToHTML(node: any): string {
+  if (!node || typeof node !== 'object') {
+    return "";
+  }
+
+  switch (node.type) {
+    case 'paragraph':
+      const pContent = node.content ? contentToHTML(node.content) : '';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)